src/web_interface/templates/show_statistic.html

```diff
@@ -198,6 +220,32 @@ <h3>{{ title }}</h3>
     {% endif %}
 {% endfor %}
 
+
+
+{# ------ Exploit Mitigation Stats ------ #}
+
+{% if stats["exploit_mitigations_stats"] %}
```

Comment:
Some of the bars are too long (> 100 %) breaking the tooltips.
Don't know where the error is.